packages/portalnetwork/src/subprotocols/history/headerAccumulator.ts

```diff
@@ -62,8 +62,7 @@ export class HeaderAccumulator {
       blockHash: newHeader.hash(),
       totalDifficulty: lastTd + BigInt(newHeader.difficulty.toString(10)),
     }
-    this._currentEpoch.push(headerRecord)
-    return this._currentEpoch.length
+    return this._currentEpoch.push(headerRecord) - 1
```

Comment:
Why subtract 1 here?  Wouldn't that mean we return 1 less than the height of the epoch?

Reply:
An epoch with 100 blocks will be at height 99, right?
I changed this to match the height, instead of the number of blocks.